Python/python.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+def rock_paper_scissors():
+    
+    turn_options = ['Rock', 'Paper', 'Scissors']
```

Comment:
stick with all lowercase here and below (and then lowercase the player's input, so it always matches)